packages/tools/kadena-cli/src/utils/globalOptions.ts

```diff
@@ -139,6 +162,27 @@ export const globalOptions = {
       }
     },
   }),
+  chainIdOptional: createOption({
```

Comment:
instead of creating this option completely - we can do like this right `globalOptions.chainId({ isOptional: true })`  and then adding a change in transform what you've added now `if (!chainId) return null;` it works, do we really need separate optional one?

Reply:
Yes because when you add `if (!chainId) return null;` it also affects the return type. I will need to go and add guards and errors everywhere that network is used.